.openshiftio/application.yaml

```diff
@@ -173,7 +173,7 @@ objects:
   metadata:
     labels:
       app: thorntail-rest-http
-      group: io.openshift.booster
+      group: io.thorntail.example
```

Comment:
Would 'io.thorntail.booster' be more specific ? Or it has to be 'io.thorntail.example' because they are in the thorntail-examples repo ?

Reply:
We're getting rid of the "booster" and "mission" words, replacing them by "example".

Reply:
OK